File: backend/src/main/java/com/godeltech/mastery/backend/service/EquipmentService.java
package com.godeltech.mastery.backend.service;

import com.godeltech.mastery.backend.domain.dto.responce.EquipmentDTO;
import com.godeltech.mastery.backend.domain.entity.Equipment;

import java.util.List;

public interface EquipmentService {

  Equipment getEquipmentById(Long id);

  List<EquipmentDTO> getEquipments();
}

File: backend/src/main/kotlin/com/godeltech/mastery/backend/service/EquipmentService.kt
package com.godeltech.mastery.backend.service

import com.godeltech.mastery.backend.domain.dto.responce.EquipmentDto
import com.godeltech.mastery.backend.domain.entity.Equipment

interface EquipmentService {

    fun getEquipmentById(id: Long) : Equipment

    fun getEquipments() : List<EquipmentDto>
}